legend-engine-xt-serviceStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/service/plugin/ServiceStoreExecutorBuilder.java

```diff
@@ -30,12 +31,18 @@ public StoreType getStoreType()
     @Override
     public ServiceStoreExecutor build()
     {
-        return ServiceStoreExecutor.INSTANCE;
+        return new ServiceStoreExecutor(new ServiceStoreState());
```

Comment:
lets call below build function with empty config instead of instantiating at multiple places